tests/framework/PostProcessors/ParetoFrontierPostProcessor/test_paretoFrontier.xml

```diff
@@ -0,0 +1,60 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors.ParetoFrontier</name>
```

Comment:
shouldn't be framework/PostProcessors/ParetoFrontierPostProcessor.ParetoFrontier?

Reply:
edited

Reply:
Fixed!